.coderabbit.yaml

```
@@ -0,0 +1,16 @@
+reviews:
+  profile: "chill"
+  request_changes_workflow: false
+  high_level_summary: true
+  poem: true
+  review_status: true
+  collapse_walkthrough: false
+  auto_review:
+    enabled: true
+    drafts: false
+chat:
+chat:
```

YAMLlint (1.37.1)

[error] duplication of key "chat" in mapping

(key-duplicates)